### ISSUE TITLE
require-default-prop warns when using vue-types named exports

### ISSUE THREAD

**Issue by art0rz** [dwightjack/eslint-plugin-vue-types#9]
`require-default-prop` warns when using `shape()`, `string()`, etc. rather than `VueTypes.shape`, `VueTypes.string`, etc.

```javascript
import VueTypes, { shape } from 'vue-types';

const MyComponent = Vue.extend({
  props: {
    data: shape({}).isRequired, // ESLint: Prop 'data' requires default value to be set.(vue-types/require-default-prop)
    data2: VueTypes.shape({}).isRequired, // this is fine
  },
});
```

**Comment by art0rz** [comment]
It also warns on this:

```javascript
const MyShape = () => VueTypes.shape({});

Vue.extend({
  props: {
    data: MyShape().isRequired, // ESLint: Prop 'data' requires default value to be set.(vue-types/require-default-prop)
  },
});
```

**Comment by dwightjack** [comment]
Hi. I think this might be related to https://github.com/dwightjack/vue-types/issues/179.

You could try installing eslint-plugin-vue-types@2.1.0-rc.1 and see if that solves the issue.

**Comment by dwightjack** [comment]
Hi, the fix has been published with `eslint-plugin-vue-types@2.1.0`. Feel free to re-open the issue if you encounter any other issues.
